src/parser/ParsedQuery.h

```diff
@@ -391,7 +391,9 @@ class ParsedQuery {
 
   ParsedQuery() = default;
 
-  vector<SparqlPrefix> _prefixes;
+  // The ql prefix for QLever specific additions is always defined.
+  vector<SparqlPrefix> _prefixes = {SparqlPrefix(
+      INTERNAL_PREDICATE_PREFIX_NAME, INTERNAL_PREDICATE_PREFIX_IRI)};
```

Comment:
I added the default prefix here instead of in `getPrefixMap` on purpose. Adding it here might lead to some confusion but it can be seen easily. Adding it in `getPrefixMap` on the other hand might be transparent in many tests. But I fear that adding it there might lead to much harder to detect bugs later on when some times the Prefixes work and other times not.